ehitchhiking-ui/src/app/shared/components/dialog/dialog.component.ts

```diff
@@ -0,0 +1,35 @@
+import {Component, OnInit} from '@angular/core';
+
+@Component({
+	selector: 'app-dialog',
+	templateUrl: './dialog.component.html',
+	styleUrls: ['./dialog.component.sass'],
+})
+export class DialogComponent implements OnInit {
```

Comment:
bad naming: DialogComponent. pls rename to DialogsListComponent

Reply:
fixed